client/README.md

```diff
@@ -149,17 +150,4 @@ func TestRedisBackendSlow(t *testing.T) {
         t.Fatal("Redis command did not take long enough:", time.Since(start))
     }
 }
-
-func TestEphemeralProxy(t *testing.T) {
-    proxy, _ := toxiClient.CreateProxy("test", "", "google.com:80")
-    defer proxy.Delete()
-
-    // Test connection through proxy.Listen
-    resp, err := http.Get("http://" + proxy.Listen)
-    if err != nil {
-        t.Fatal(err)
-    } else if resp.StatusCode != 200 {
-        t.Fatal("Proxy to google failed:", resp.StatusCode)
-    }
-}
```

Comment:
Since we're planning on cleaning up this test, we could probably get rid of the `net/http` import as well since this is the only test that uses it.